GooglePlacesAutocomplete.js

```diff
@@ -565,24 +604,11 @@ const GooglePlacesAutocomplete = React.createClass({
           automaticallyAdjustContentInsets={false}
           {...this.props}
           renderRow={this._renderRow}
+          renderFooter={this._renderPoweredLogo}
         />
       );
     }
 
-    if(this.props.enablePoweredByContainer) {
```

Comment:
ok, I'll revert it back if you'd like, but I think that flag `enablePoweredByContainer` should be `false` by default then, otherwise 2 same images would be displayed.